src/axom/quest/ArrayIndexer.hpp

```diff
@@ -10,8 +10,19 @@
 #include "axom/core/StackArray.hpp"
 #include "axom/core/numerics/matvecops.hpp"
 
+#include <iostream>
+#include <sstream>
```

Comment:
In the interface below, it's not clear to me how to provide the shape info via the StackArray. Perhaps, a description at the top (rather than repeating for each call) or a reference to some other documentation where it is explained would help.

Reply:
Agreed.  Thanks.